.github/workflows/lint.yml

```diff
@@ -3,7 +3,7 @@
 # these after push is meaningless to the building of the package
 name: python lint
 
-on: [pull_request]
+on: [pull_request, push]
```

Comment:
I never used this behaviour. I find it not explicit and honestly don't know what it does. I would use instead:

```yaml
on:
  pull_request:
    branches: [master]
```

You don't want `lint` to trigger when the PR is accepted. And likely you don't want GHA to trigger at all when Pull Requesting from one branch to another that is not the `main`/`master`.

Reply:
good point!